.github/workflows/swaggerhub-delete.yml

```diff
@@ -0,0 +1,26 @@
+name: Delete spec version from SwaggerHub
+
+on:
+  [delete]
+
+jobs:
+  removespec:
+    name: Remove the corresponding spec from SwaggerHub
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: set output for versioning
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
```

Comment:
Probably we should use here `'` to prevent substitution of `*` with glob?

Reply:
Right. Added double quotes now.